src/cpp/multisolver_interface/SolverXpress.cpp

```diff
@@ -6,6 +6,8 @@
 
 #include "StringManip.h"
 
+using namespace operations_research_Xpansion;
```

Comment:
`operations_research` is kind of dedicated to OR-Tools. I recommend choosing a better name here

Reply:
Merci